pkg/cri/server/container_create_linux.go

```diff
@@ -55,6 +55,9 @@ const (
 	unconfinedProfile = "unconfined"
 	// seccompDefaultProfile is the default seccomp profile.
 	seccompDefaultProfile = dockerDefault
+
+	runtimeEnableLegacyCgroupNS = "vke.volcengine.com/runtime-enable-legacy-cgroupns"
```

Comment:
sorry，it's your platform feature. I don't think we should accept it.

Reply:
This is not the code that should be pushed. I have updated this.
```go
// ContainerCgroupNamespace is the sandbox annonation for running container in new cgroup namespace.
ContainerDisableCgroupNamespace = "io.kubernetes.cri.disable-container-cgroupns"
```